src/main/java/frc/robot/robotcontainers/ContainerForTesting.java

```diff
@@ -13,6 +13,7 @@
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.Commands;
 import edu.wpi.first.wpilibj2.command.ScheduleCommand;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
```

Comment:
Please delete these changes in ContainerForTesting.java. 
You will use NewRobotContainer.java